src/style/colors.ts

```diff
@@ -15,7 +15,11 @@ export type DesignToken = {semantic?: [string]; resource_paths?: [string]; toStr
 export type TokensOptions = {primaryColor: string};
 export type GetColorTintOptions = {avoidReverseOnDark?: boolean};
 export type GetColorByHexOptions = {validColors?: string[]};
-
+export type PaletteOptions = {
```

Comment:
Rename to GeneratePaletteOptions so it's more associated with the method

Reply:
Done